### ISSUE TITLE
dbic_connect_attrs is ignored when populating data in DBIx::Class::Migration::RunScript

### ISSUE THREAD

**Issue by abeverley** [jjn1056/DBIx-Class-Migration#63]
I am seeding data in my database as per this: https://metacpan.org/pod/distribution/DBIx-Class-Migration/lib/DBIx/Class/Migration/Tutorial/FirstMigration.pod#Customizing-the-Migration

However, dbic_connect_attrs is not being used when autogenerating the DBIx::Class::Schema::Loader schema, which means it is not possible to seed data for table names that require quoting (`quote_names` is dropped).

For info, I am using the following as a workaround:

```
use DBIx::Class::Migration::RunScript;
migrate {
    my $schema = shift->schema;
    $schema->storage->connect_info([sub {$schema->storage->dbh}, { quote_names => 1 }]);
    ...
}
```

**Comment by abeverley** [comment]
For anyone looking at this, this is the offending piece of code: https://github.com/jjn1056/DBIx-Class-Migration/blob/master/lib/DBIx/Class/Migration/RunScript/Trait/SchemaLoader.pm#L13

Only $dbh is passed to make_schema_at(), not any other options.

**Comment by mohawk2** [comment]
I believe this is now unnecessary, at least for identifier-quoting, due to the fix for #66. If you can be more specific about a repro case, I can look again. Closing in meantime.
